_inc/lib/tracks/tracks-ajax.js

```diff
@@ -0,0 +1,52 @@
+/* global jpTracksAJAX, jQuery */
+
+(function( $, jpTracksAJAX ) {
+
+	var data;
+
+	$( document ).ready( function () {
+
+		data = {
+			'tracksNonce' : jpTracksAJAX.jpTracksAJAX_nonce
+		};
+
+		jetpackTracksAJAX();
+	});
+
+	function jetpackTracksAJAX() {
+		$( '.jptracks' ).click( function( e ) {
+			data.action           = 'jetpack_tracks';
+			data.tracksEventType  = 'click';
+			data.tracksEventName  = $( this ).attr( 'data-jptracks-name' );
+			data.tracksEventProp  = $( this ).attr( 'data-jptracks-prop' ) || false;
+
+			// We need an event name at least
+			if ( undefined === data.tracksEventName ) {
+				return;
+			}
+
+			e.preventDefault();
+
+			var url    = e.srcElement.href;
+			var target = e.srcElement.target;
+			if ( url && target && '_self' !== target ) {
+				var newTabWindow = window.open( '', target );
+			}
+
+			$.post( jpTracksAJAX.ajaxurl, data, function ( response ) {
+				if ( response.success ) {
+
+					// Continue on to whatever url they were trying to get to.
+					if ( url ) {
```

Comment:
> we may lose a significant part of the stats because the browser would cancel any async requests on the unload event.

That was my initial concern, but then doubted myself.  Thanks for the sanity check. 
True about not needing to wait for success.  `always()` seems fine